clarity/src/vm/ast/parser/v1.rs

```diff
@@ -112,26 +112,32 @@ fn get_lines_at(input: &str) -> Vec<usize> {
     out
 }
 
-lazy_static! {
-    pub static ref STANDARD_PRINCIPAL_REGEX: String =
-        "[0123456789ABCDEFGHJKMNPQRSTVWXYZ]{28,41}".into();
-    pub static ref CONTRACT_NAME_REGEX: String = format!(
+pub static STANDARD_PRINCIPAL_REGEX: LazyLock<String> =
+    LazyLock::new(|| "[0123456789ABCDEFGHJKMNPQRSTVWXYZ]{28,41}".into());
+pub static CONTRACT_NAME_REGEX: LazyLock<String> = LazyLock::new(|| {
+    format!(
```

Comment:
I think it would be better to use the `const_format` crate ([link](https://crates.io/crates/const_format)) for format strings. It's cleaner and results in  constant `&str`s that don't require locking to access

Reply:
I'm not familiar with that crate, but this PR is only refactoring away `lazy_static` and not changing anything else.

How would you use that crate here? The highlighted code is constructing a `Regex` static lazily, not formatting.

Reply:
Ah nevermind, I'm looking on my phone and it was a bit hard to see -- I think I see what you mean, I'll try it.

Reply:
Adding `const_format` could be done in a separate PR, but there are places where the use of either `lazy_static!()` or `LazyLock` is unnecessary, and that should be addressed in this PR

Reply:
Yeah, my goal was to simply remove the lazy_static dep with as little change as possible, but i'm happy to look at further refactorings. Will go through all your comments this weekend 👍